modules/installation/pages/install-proxy-uyuni.adoc

```diff
@@ -0,0 +1,52 @@
+
+USE IT AS TEMPLATE FOR PROXY INSTALL
+
+
+[[install-proxy-uyuni]]
+= Install Uyuni Proxy with openSUSE
```

Comment:
FMPOV we should you use `{productname}`, {opensuse}, etc, everywhere. Same comment for the rest of the files.

Reply:
Unfortunately, it is not that clear whether macros in titles will work reliably, @jcayouette @Loquacity ? At least, in the past, we had issues...

Reply:
They fail if it's the first word, from my testing.

Reply:
ok, I think I adjusted it everywhere where it will work.